satellite/rewards/rewards.go

```diff
@@ -57,6 +57,8 @@ const (
 	FreeCredit = OfferType(1)
 	// Referral is a type of offers used for Referral Program
 	Referral = OfferType(2)
+	// Partner is a type of offers used for Open Source Partner Program
+	Partner = OfferType(3)
```

Comment:
why we don't use string for status? it is easier to maintain this way

Reply:
Good point. I think we started with integer so we just kept going this route. Egon also mentioned that a string would be better. I think it's worth to be another PR to make this change